wcc-business-executives/WCC Example.ipynb

```diff
@@ -0,0 +1,474 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "39835177-2c76-45c9-9d78-ade7c7efc0cb",
+   "metadata": {
+    "collapsed": false,
+    "name": "cell1"
+   },
+   "source": [
+    "Author: Dave Fauth\n",
+    "\n",
+    "Last Updated: 7 May 2025\n",
+    "\n",
+    "Neo4j Graph Analytics for Snowflake Version: 0.3.13"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60afe040-12dc-4554-a747-e1ba04a7e13f",
+   "metadata": {
+    "collapsed": false,
+    "name": "cell2"
+   },
+   "source": [
+    "The Weakly Connected Components (WCC) algorithm finds sets of connected nodes in directed and undirected graphs. Two nodes are connected, if there exists a path between them. The set of all nodes that are connected with each other form a component.\n",
+    "\n",
+    "This example uses data that was sourced from `Relationship 360 - People Business Intelligence Sample (Massachusetts)`. This data set is a free dataset that is available in the Snowflake Marketplace."
```

Comment:
Probably it's useful to have a link here

Is it a right link?
https://app.snowflake.com/marketplace/listing/GZ2FQZ711VO/equilar-relationship-360-people-business-intelligence-sample-massachusetts